src/plugins/intel_cpu/src/itt.h

```diff
@@ -4,18 +4,20 @@
 
 /**
  * @brief Defines openvino domains for tracing
- * @file mkldnn_itt.h
+ * @file itt.h
  */
 
 #pragma once
 
 #include <openvino/itt.hpp>
 
-namespace MKLDNNPlugin {
+namespace ov {
+namespace intel_cpu {
 namespace itt {
 namespace domains {
-    OV_ITT_DOMAIN(MKLDNNPlugin);
+    OV_ITT_DOMAIN(intel_cpu);
     OV_ITT_DOMAIN(MKLDNN_LT);
```

Comment:
Shouldn't we rename this domain as well?

Reply:
done